### ISSUE TITLE
Submit listen without release_name(album)

### ISSUE THREAD

**Issue by MinmoTech** [InputUsername/rescrobbled#83]
I get the following error for most of the songs I downloaded locally from youtube:
```
Failed to update status on ListenBrainz

Caused by:
    API error (400): field track_metadata.release_name is empty.
Failed to submit track to ListenBrainz

Caused by:
    API error (400): field track_metadata.release_name is empty.
```
The [listenbrains documentation](https://listenbrainz.readthedocs.io/en/latest/users/json.html#payload-json-details) lists:
![image](https://user-images.githubusercontent.com/25890405/199020715-06d7e8a2-b3d6-4f43-a33a-f7b3cca3bc8b.png)

Which means the release_name field should be optional.

I assume the issue is that release_name gets submitted with an empty value while it should probably be omitted entirely instead.

Thanks for the awesome project and any kind of support!

**Comment by InputUsername** [comment]
Thanks for reporting this! This is a bug/limitation in my ListenBrainz library, could you please make an issue on its repo? https://github.com/InputUsername/listenbrainz-rs

>Thanks for the awesome project and any kind of support!

Glad you find it useful!

**Comment by InputUsername** [comment]
@MinmoTech I've just released 0.7.0 which makes the album optional when submitting to ListenBrainz. Please let me know if it works for you so we can close this issue :smile:

**Comment by MinmoTech** [comment]
This works really well now!

I really need to set up a filter script so I don't pollute my listens too much now :grinning: 

Thanks a lot!
